lib/metro-transit.js

```diff
@@ -0,0 +1,106 @@
+    response.tellWithCard(props.helpSpeechOutput, props.helpCardTitle, props.helpCardText);
+  },
+
+  GetStation: function (intent, session, response) {
+    var stationName = utils.changeStationName(intent.slots.station.value, 'correction');
+    console.info('Station Name: ' + stationName);
+
+    if (_.has(stations, stationName)) {
+      var stationCode = stations[stationName].Code;
+
+      WmataApi.get(props.stationArrivalWmataUrl + stationCode, response, function(body) {
+        var trainArrivals = _.reduce(body.Trains, function(result, train) {
+          if (train.DestinationName === 'Train' || train.DestinationName == 'No Passenger') return result;
+
+          var arrivals = result[train.DestinationName] || [];
+          arrivals.push(train.Min);
+          result[utils.changeStationName(train.DestinationName, 'abbreviation').toLowerCase()] = arrivals;
+          return result;
+        }, {});
+
+        console.info('Train Arrivals', trainArrivals);
+        if (_.size(trainArrivals) === 0) {
+          response.tell(props.stationNoArrivalsSpeechOutput);
+        } else {
+          var destinationNeededText = 'Are you going to ' + utils.joinListConjuction(_.keys(trainArrivals), ', ', ' or ');
+          session.attributes = trainArrivals;
+          response.ask(destinationNeededText, destinationNeededText);
+        }
+      });
+    }
+  },
+
+  GetDestinationStation: function (intent, session, response) {
+    var sessionAttributes = session.attributes;
+    var destinationStationName = intent.slots.destinationStation.value;
+    console.info('Station Name: ' + destinationStationName);
+
+    if (_.has(sessionAttributes, destinationStationName)) {
+      var arrivalTimes = sessionAttributes[destinationStationName];
+      var respText = 'The next ' + (arrivalTimes.length === 1 ?
+        'train' : arrivalTimes.length + ' trains') + ' heading to ' + destinationStationName +
+        (arrivalTimes.length === 1 ? ' arrives' : ' arrive') + ' in ' + utils.joinListConjuction(arrivalTimes, ', ', ' and ') + ' minutes.';
+      response.tell(respText);
+    } else {
+      console.error(props.destinationNotFoundConsole + ' ' + destinationStationName);
+      response.tell(props.destinationNotFoundSpeechOutput + ' ' + destinationStationName + '.');
+    }
+  },
+
+  GetServiceAdvisories: function(intent, session, response) {
+    WmataApi.get(props.serviceAdvisoriesWmataUrl, response, function(body) {
+      var incidents = body.Incidents;
+      if (!incidents) {
+        console.error(props.serviceAdvisoriesErrorConsole);
+        response.tell(props.serviceAdvisoriesErrorSpeechOutput);
+        return;
+      }
+      var incidentList = _.reduce(incidents, function (descriptions, incident) {
+        return incident.Description ? descriptions.push(incident.Description) : descriptions;
+      }, []);
+      response.tell(incidentList.join('\n'));
+    });
+  }
+};
+
+module.exports = new MetroTransit();
```

Comment:
That is what I was intending. Is there a reason we should not make it a singleton? Both ways are totally valid. Think of it like Java. Is a singleton needed/wanted vs regular instances? My thought was we don't want to make more instances, ever. Right now, our code only uses it once, so I think it doesn't matter.